packages/fiori/src/themes/TimelineItem.css

```diff
@@ -44,6 +82,16 @@
 	transform: translateX(-50%);
 }
 
+/* No Icon Dot in Hotizontal */
+:host([axis-orientation="Horizontal"]:not([icon])) .ui5-tli-indicator::after {
+	top: 0.78125rem;
```

Comment:
There are quite a lot wild numbers in this css file, I would need more time to take a closer look and perhaps test it manually. Maybe others from your team can check if this is fine:

```css
top: 0.78125rem;
left: 0.9625rem;

left: 42%;
width: 75%;
transform-origin: 152% 104%;

right: 1.9375rem;
left: -1.625rem;
	
left: -1.625rem;
right: 2.125rem;
```

Reply:
I am looking into this change as well